src/main/webapp/WEB-INF/pages/inc/menu.ftl

```diff
@@ -98,11 +98,6 @@
                       </a>
```

Comment:
I think the health icon and its link (http://localhost:8081/health.do) in the navbar-nav div cause the misbehavior when you click on heartbeat button. 
![Screenshot_20210728_104727](https://user-images.githubusercontent.com/72577339/127354064-257e6aea-83e4-4ef6-8511-b14d1b7cc7f2.png)
What is the purpose health.do ?  or Can we just delete it?